package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=16"
   },
-  "version": "8.1.161",
+  "version": "8.1.162",
```

Comment:
not sure we want to bump just yet: we may want to wait for the other related changes for putObjectVerCase2 / ... (if possible, c.f. TAD) and the versionID uniqueness improvement